docs/tutorials/polynome2/problem_step_1.py

```diff
@@ -3,10 +3,10 @@
 Problem = HpProblem()
 
 Problem.add_dim("units", (1, 100))
-Problem.add_dim("activation", [None, "relu", "sigmoid", "tanh"])
```

Comment:
Currently the value is directly passed to the `run` function therefore it should be mapped to `None` before being passed to the `keras.Dense(...)`.

Also we should probably change the use of `add_dim` in the tutorial to `Problem.add_hyperparameter` to be more consistent with the 2 other HP tutorials (`add_dim` is deprecated).